solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -1957,6 +1966,19 @@ public void unload(String name, boolean deleteIndexDir, boolean deleteDataDir, b
     if (close)
       core.closeAndWait();
 
+    if(coll != null && coll.isPerReplicaState()) {
```

Comment:
I know it may seem trivial, but can we please default to using if-statement whitespace conventions that are idiomatic to the rest of the lucene-solr codebase?

It actually _isn't_ strictly trivial -- there's a real practical consequence to this in the sense that it's virtually guaranteed to generate spurious merge conflicts when updating locally, if analogous changes are merged upstream (where they will no doubt use standard whitespace conventions).